src/main/java/com/faforever/client/play/PlayController.java

```diff
@@ -61,15 +76,21 @@ protected void onDisplay(NavigateEvent navigateEvent) {
     try {
       if (Objects.equals(navigateEvent.getClass(), NavigateEvent.class)
           || navigateEvent instanceof OpenCustomGamesEvent) {
-        playRootTabPane.getSelectionModel().select(customGamesTab);
+        Node node = customGamesController.getRoot();
```

Comment:
I get an exception on this line. Any idea what the problem is? Is the customGamesController not initialied yet?